spec/unit/facter/postfix_version_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Facter::Util::Fact.to_s do
```

Comment:
You're actually testing a specific fact, so I'd rather use:

```suggestion
describe Facter.fact(:postfix_version) do
```